Cargo.toml

```diff
@@ -36,26 +36,26 @@ default = [
     "yamux",
 ]
 autonat = ["libp2p-autonat"]
-dcutr = ["libp2p-dcutr", "libp2p-metrics/dcutr"]
+dcutr = ["libp2p-dcutr", "libp2p-metrics?/dcutr"]
 deflate = ["libp2p-deflate"]
 dns-async-std = ["libp2p-dns", "libp2p-dns/async-std"]
 dns-tokio = ["libp2p-dns", "libp2p-dns/tokio"]
 floodsub = ["libp2p-floodsub"]
-identify = ["libp2p-identify", "libp2p-metrics/identify"]
-kad = ["libp2p-kad", "libp2p-metrics/kad"]
-gossipsub = ["libp2p-gossipsub", "libp2p-metrics/gossipsub"]
+identify = ["dep:libp2p-identify", "libp2p-metrics?/identify"]
+kad = ["libp2p-kad", "libp2p-metrics?/kad"]
+gossipsub = ["libp2p-gossipsub", "libp2p-metrics?/gossipsub"]
 metrics = ["libp2p-metrics"]
 mdns = ["libp2p-mdns"]
 mplex = ["libp2p-mplex"]
 noise = ["libp2p-noise"]
-ping = ["libp2p-ping", "libp2p-metrics/ping"]
+ping = ["dep:libp2p-ping", "libp2p-metrics?/ping"]
 plaintext = ["libp2p-plaintext"]
 pnet = ["libp2p-pnet"]
-relay = ["libp2p-relay", "libp2p-metrics/relay"]
+relay = ["dep:libp2p-relay", "libp2p-metrics?/relay"]
 request-response = ["libp2p-request-response"]
 rendezvous = ["libp2p-rendezvous"]
-tcp-async-io = ["libp2p-tcp", "libp2p-tcp/async-io"]
-tcp-tokio = ["libp2p-tcp", "libp2p-tcp/tokio"]
+tcp-async-io = ["dep:libp2p-tcp", "libp2p-tcp?/async-io"]
+tcp-tokio = ["dep:libp2p-tcp", "libp2p-tcp?/tokio"]
```

Comment:
I think here, the `?` is unnecessary? We are activating a feature on the same dependency that we are activating.

Reply:
Good catch, I overlooked this.